Pipelines/test-execution-steps.yml

```diff
@@ -0,0 +1,122 @@
+parameters:
+  - name: IsMultiRegionEnabled
+    type: boolean
+    default: false
+
+  - name: RegionNumber
+    type: number
+
+  - name: IsClusterRequired
+    type: boolean
+    default: false
+
+  - name: JmeterFolderPath
+    type: string
+    default: JMeterFiles
+
+  - name: JmeterFileName
+    type: string
+    default: SampleTestPlan.jmx
+
+  - name: Threads
+    type: number
+    default: 10
+
+  - name: Duration
+    type: number
+    default: 300
+
+  - name: Loops
+    type: number
+    default: 3
+
+  - name: RampUpTime
+    type: number
+    default: 10
+
+  - name: Tenant
+    type: string
+
+  - name: Namespace
+    type: string
+
+  - name: ServiceConnection
+    type: string
+
+  - name: KeyVaultName
+    type: string
+
+  - name: SecretNames
```

Comment:
can we keep it as SecretName ? Is it array ?

Reply:
yes its an array, you can pass comma separated values